src/main/java/hudson/plugins/locale/PluginImpl.java

```diff
@@ -52,7 +55,15 @@ protected void load() throws IOException {
     }
 
     protected XmlFile getConfigXml() {
-        return new XmlFile(XSTREAM, new File(Hudson.getInstance().getRootDir(),"locale.xml"));
+        return new XmlFile(XSTREAM, new File(Jenkins.getActiveInstance().getRootDir(),"locale.xml"));
+    }
+
+    public String getDescription() {
+        return Messages.description();
```

Comment:
🐛 Here you expose configuration option description to the plugin description. I doubt it is a good idea, will complicate extending the plugin